api/src/test/java/uk/gov/dhsc/htbhf/claimant/testsupport/EntityAgeAcceleratorTest.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.dhsc.htbhf.claimant.testsupport;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.dhsc.htbhf.claimant.entity.PaymentCycle;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.*;
+import static uk.gov.dhsc.htbhf.claimant.testsupport.PaymentCycleTestDataFactory.aValidPaymentCycleBuilder;
+
+@ExtendWith(MockitoExtension.class)
+class EntityAgeAcceleratorTest {
+
+
+    @Test
+    void shouldFastForwardPaymentCycle() {
+
+        final LocalDateTime originalCardBalanceTimestamp = LocalDateTime.now();
+        PaymentCycle paymentCycle = aValidPaymentCycleBuilder()
+                .cardBalanceTimestamp(originalCardBalanceTimestamp)
+                .build();
+        final LocalDate originalStartDate = paymentCycle.getCycleStartDate();
+        final List<LocalDate> originalChildrenDob = paymentCycle.getChildrenDob();
+        final LocalDateTime originalClaimStatusTime = paymentCycle.getClaim().getClaimStatusTimestamp();
+        final LocalDate originalEntitlementDate = paymentCycle.getVoucherEntitlement().getVoucherEntitlements().get(0).getEntitlementDate();
+
+        EntityAgeAccelerator.fastForward(paymentCycle, 28);
+
+        assertThat(paymentCycle.getCycleStartDate()).isEqualTo(originalStartDate.minusDays(28));
+        assertThat(paymentCycle.getCardBalanceTimestamp()).isEqualTo(originalCardBalanceTimestamp.minusDays(28));
+        assertThat(paymentCycle.getChildrenDob().get(0)).isEqualTo(originalChildrenDob.get(0));
```

Comment:
Why are we asserting that the children's dates of birth are unchanged? They should have been mutated.